index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>personal portfolio</title>
+</head>
+
+<body>
+    <div class="container">
```

Comment:
- [ ] It would be better if we change the `<div>` tag to `<section>` tag to improve HTML semantics. This helps in SEO optimization and also helps aid communication

Reply:
thanks but the div is just to house the flex property around the whole content, every children element below followed html semantics, I just replicated the instruction I saw on the recommended flex video i watched